config/runtime.exs

```diff
@@ -17,3 +17,22 @@ config :ueberauth, Ueberauth.Strategy.Cognito,
   client_secret: System.get_env("COGNITO_CLIENT_SECRET"),
   user_pool_id: System.get_env("COGNITO_USER_POOL_ID"),
   aws_region: System.get_env("COGNITO_AWS_REGION")
+
+if config_env() == :dev do
```

Comment:
question: is there a reason this shouldn't live in `config/dev.exs`?

Reply:
See https://github.com/mbta/document_viewer/pull/403#discussion_r1815619479